### ISSUE TITLE
Import latest mori to benchmark against

### ISSUE THREAD

**Issue by hughfdjackson** [hughfdjackson/persistent-hash-trie#23]
To establish the performance difference between mori and this library, bring it in for benchmarking :D

**Comment by hughfdjackson** [comment]
Current version vs mori: 

``` javascript

assoc property with Trie of 1
//-------------------------//
shallow-copy reference  : 10,019,767.15 ± 6.76% ops/sec
v0.2.1                  : 789,608.85 ± 0.58% ops/sec
v0.3.1                  : 790,450.91 ± 0.93% ops/sec
mori : 3,489,822.22 ± 1.01% ops/sec

assoc property with Trie of 10
//-------------------------//
shallow-copy reference  : 225,094.57 ± 0.87% ops/sec
v0.2.1                  : 341,698.71 ± 1.72% ops/sec
v0.3.1                  : 348,841.06 ± 1.90% ops/sec
mori : 2,572,287.52 ± 0.69% ops/sec

assoc property with Trie of 100
//-------------------------//
shallow-copy reference  : 28,148.25 ± 1.52% ops/sec
v0.2.1                  : 115,403.87 ± 1.84% ops/sec
v0.3.1                  : 128,135.10 ± 1.56% ops/sec
mori : 2,007,885.81 ± 2.11% ops/sec

assoc property with Trie of 1000
//-------------------------//
shallow-copy reference  : 2,550.86 ± 6.62% ops/sec
v0.2.1                  : 103,028.21 ± 0.56% ops/sec
v0.3.1                  : 63,717.09 ± 1.83% ops/sec
mori : 1,515,967.15 ± 3.28% ops/sec

dissoc property with Trie of 1
//-------------------------//
shallow-copy reference  : 605,582.54 ± 2.19% ops/sec
v0.2.1                  : 515,574.39 ± 1.82% ops/sec
v0.3.1                  : 384,564.27 ± 0.82% ops/sec
mori : 2,426,483.65 ± 0.80% ops/sec

dissoc property with Trie of 10
//-------------------------//
shallow-copy reference  : 212,925.58 ± 1.13% ops/sec
v0.2.1                  : 175,516.52 ± 2.06% ops/sec
v0.3.1                  : 142,042.56 ± 1.68% ops/sec
mori : 2,208,260.53 ± 0.93% ops/sec

dissoc property with Trie of 100
//-------------------------//
shallow-copy reference  : 24,247.93 ± 0.69% ops/sec
v0.2.1                  : 100,602.73 ± 1.10% ops/sec
v0.3.1                  : 87,260.78 ± 1.16% ops/sec
mori : 2,110,692.67 ± 0.80% ops/sec

dissoc property with Trie of 1000
//-------------------------//
shallow-copy reference  : 3,012.81 ± 0.67% ops/sec
v0.2.1                  : 51,961.21 ± 1.05% ops/sec
v0.3.1                  : 38,783.51 ± 0.94% ops/sec
mori : 1,593,777.70 ± 1.28% ops/sec

has property with Trie of 1
//-------------------------//
shallow-copy reference  : 10,584,346.48 ± 1.78% ops/sec
v0.2.1                  : 10,464,541.01 ± 1.43% ops/sec
v0.3.1                  : 10,011,291.63 ± 1.15% ops/sec
mori : 4,132,145.91 ± 17.25% ops/sec

has property with Trie of 10
//-------------------------//
shallow-copy reference  : 2,020,002.73 ± 3.40% ops/sec
v0.2.1                  : 1,399,607.89 ± 4.70% ops/sec
v0.3.1                  : 438,341.62 ± 4.33% ops/sec
mori : 1,346,568.15 ± 1.13% ops/sec

has property with Trie of 100
//-------------------------//
shallow-copy reference  : 2,354,790.77 ± 1.38% ops/sec
v0.2.1                  : 1,158,857.85 ± 1.18% ops/sec
v0.3.1                  : 1,196,021.58 ± 2.42% ops/sec
mori : 1,494,502.19 ± 2.78% ops/sec

has property with Trie of 1000
//-------------------------//
shallow-copy reference  : 2,292,416.63 ± 1.55% ops/sec
v0.2.1                  : 771,815.79 ± 1.61% ops/sec
v0.3.1                  : 809,081.31 ± 1.77% ops/sec
mori : 1,568,352.31 ± 0.92% ops/sec

get property with Trie of 1
//-------------------------//
shallow-copy reference  : 8,223,409.00 ± 0.92% ops/sec
v0.2.1                  : 2,588,647.33 ± 1.69% ops/sec
v0.3.1                  : 2,498,843.76 ± 0.96% ops/sec
mori : 1,715,720.58 ± 1.66% ops/sec

get property with Trie of 10
//-------------------------//
shallow-copy reference  : 8,419,171.56 ± 1.63% ops/sec
v0.2.1                  : 2,567,683.09 ± 1.91% ops/sec
v0.3.1                  : 2,613,825.49 ± 1.05% ops/sec
mori : 1,635,737.55 ± 2.82% ops/sec

get property with Trie of 100
//-------------------------//
shallow-copy reference  : 8,113,544.13 ± 2.13% ops/sec
v0.2.1                  : 986,631.84 ± 1.64% ops/sec
v0.3.1                  : 968,207.85 ± 1.92% ops/sec
mori : 1,438,726.28 ± 2.37% ops/sec

get property with Trie of 1000
//-------------------------//
shallow-copy reference  : 7,359,753.77 ± 0.92% ops/sec
v0.2.1                  : 621,985.63 ± 1.67% ops/sec
v0.3.1                  : 700,142.60 ± 1.80% ops/sec
mori : 901,809.42 ± 1.80% ops/sec

```

**Comment by hughfdjackson** [comment]
Closed by 67b07f9
